### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Comandeer**

### CODE REVIEW

**Review by KarolDawidziuk — CHANGES_REQUESTED** [review]
Hi @Comandeer 
There are a few packages that we can update, especially the `typescript`, let's update it and we'll be ready to merge.

File: package.json [hunk 1]
```diff
@@ -55,49 +55,49 @@
...
+		"jasmine": "^4.5.0",
+		"karma": "^6.4.1",
 		"karma-babel-preprocessor": "^8.0.2",
 		"karma-chrome-launcher": "^3.1.1",
 		"karma-jasmine": "^5.1.0",
 		"karma-mocha-reporter": "^2.2.5",
 		"karma-rollup-preprocessor": "^7.0.8",
-		"minimist": "^1.2.6",
+		"minimist": "^1.2.8",
 		"nightwatch": "2.0.10",
```
[the remark this answers is not in the document]
Comment (Comandeer) [reply package.json:89]: We can't update `nightwatch` until #287 is fixed.

File: package.json [hunk 2]
```diff
@@ -55,49 +55,49 @@
...
+		"rollup-plugin-polyfill-node": "^0.10.2",
 		"rollup-plugin-progress": "^1.1.2",
 		"rollup-plugin-terser": "^7.0.2",
-		"semver": "^7.3.7",
+		"semver": "^7.3.8",
 		"shelljs": "^0.8.5",
 		"tree-kill": "^1.2.2",
-		"tslib": "^2.4.0",
+		"tslib": "^2.5.0",
 		"typescript": "~4.7.4"
```
[the remark this answers is not in the document]
Comment (Comandeer) [reply package.json:101]: I would be cautious about updating TS as they don't use semver – there are probably some breaking changes. But I'll check that.

**Comment by Comandeer** [comment]
@KarolDawidziuk, thx for the review! I've bumped the deps, except the `nighwatch` (which is blocked by #287).

**Review by KarolDawidziuk — APPROVED** [review]
Thanks, @Comandeer for the improvements.

LGTM! 🎉 

There is one failing test that we should check. It's can be related to an outdated version of the `nightwatch` package, so we should check if fixing https://github.com/ckeditor/ckeditor4-react/issues/287 will help.

**Pull request merged**